web/webpack.config.js

```diff
@@ -109,11 +110,6 @@ const config = {
     new HtmlWebpackPlugin({
       minify: { removeComments: true },
       template: 'src/index.html'
-    }),
-    new HtmlWebpackTagsPlugin({
-      tags: ['app.css'],
-      append: true,
-      hash: true
```

Comment:
Previously we relied on this `HtmlWebpackTagsPlugin` to add a query parameter to the CSS tags (e.g., `<link href="app.css?ca22bf35295a91c36fb0" rel="stylesheet">`), but with the `MiniCssExtractPlugin` adding hashes directly to the filename, we don't need the query parameter anymore, so we can ditch this plugin altogether.